PENDING.md

```diff
@@ -31,6 +31,7 @@ FEATURES
 * [cosmos-sdk-cli] Added support for cosmos-sdk-cli tool under cosmos-sdk/cmd	
    * This allows SDK users to initialize a new project repository.
 * [tests] Remotenet commands for AWS (awsnet)
+* [networks] Added ansible scripts to upgrade seed nodes on a network
```

Comment:
Was this created from an existing issue? I believe the consensus is to add the issue # here, but if there is none, that's okay :-)

Reply:
No, there was no issue associated with this - at least not in the cosmos-sdk repo. There might be one in the devops repo but that's a private repo. Adding the issue number here from there wouldn't help the community at all.
I think the correct solution is to open issues on the cosmos-sdk repo regarding infrastructure scripts - which is additional administrative overhead, but doable.